code/game/objects/items/weapons/defib.dm

```diff
@@ -381,6 +381,8 @@
 								busy = 0
 								update_icon()
 								return
+					if(H.heart_attack)
+						H.heart_attack = 0
```

Comment:
I think you can only defib dead people. If it's getting a use that can be used on alive people too, it needs a little reworking.